### PR TITLE
[INT-2533] Support TS 3.9.2

### PR DESCRIPTION
**Pull request by agataHappy**
https://dronedeploy.atlassian.net/browse/INT-2533

### CODE REVIEW

**Comment by agataHappy** [comment]
Do we also want to update `tslint` and `lint-staged`?

File: package.json [hunk 1]
```diff
@@ -16,6 +16,6 @@
     "lint-staged": "^7.2.0",
     "prettier": "^1.19.1",
     "tslint": "^5.12.1",
-    "typescript": ">=3.3.x =<3.5.x "
+    "typescript": ">=3.3.x =<3.9.x "
```
Comment (s992) [inline package.json]:
```suggestion
    "typescript": ">=3.3.x <=3.9.x "
```

**Review by s992 — CHANGES_REQUESTED** [review]

**Comment by s992** [comment]
@agataHappy 

> Do we also want to update tslint and lint-staged?

tslint is deprecated, so probably not worth the effort to upgrade it. We'll likely switch to eslint in the future, and can also look at bumping lint-staged at that point.

**Comment by agataHappy** [comment]
@s992 Thanks for replay. I commited your suggestion

**Review by s992 — APPROVED** [review]

**Pull request merged**